### DISCUSSION TITLE
Setup Energydashboard using only the Senec WebAPI

### DISCUSSION THREAD

**Discussion by Basti-Fantasti** [marq24/ha-senec-v3#114] [Q&A]
Hi all,

after getting my Senec 2.1 finally exchanged with a Home P4 I have no local API anymore which could be polled for proper statistics.
How to setup the Energydashboard using the Webapi? 
I set the values in the dashboard as shown below, but nothing gets recorded.

![grafik](https://github.com/user-attachments/assets/355c5c1f-2d7e-436b-bfbb-3d2f7e20ba16)

Any idea what I made wrong?
Thanks in advance.

Best regards
Bastian

**Comment by marq24** [comment]
Just see as reference: https://github.com/marq24/ha-senec-v3/issues/11
